platform/ios/jazzy.yml

```diff
@@ -23,6 +23,7 @@ custom_categories:
       - Working with GeoJSON Data
       - For Style Authors
       - Info.plist Keys
+      - Using Gesture Recognizers
```

Comment:
The name of the .md file must match the entry in this list verbatim. Rename the file “Gesture Recognizers.md” and update this line.